### PR TITLE
feature #94 mltc benchmark mock

### PR DESCRIPTION
**Pull request by duke-b**
# Description

### Issue

This PR addresses issue #[issue-number].

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/DevoteamNL/opendora/blob/main/CONTRIBUTING.md).**
- [ ] **I have ran the linting and formatting scripts to make sure the code is consistently styled.**
- [x] **I have built/compiled and ran the tests to make sure the code works correctly.**
- [x] **I have added or changed tests to cover new or different functionality, where applicable.**
- [ ] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### CODE REVIEW

File: backstage-plugin/plugins/open-dora/src/locales/en/translation.json [hunk 1]
```diff
@@ -9,7 +9,11 @@
       "lt-6month": "Fewer than once per six months",
       "on-demand": "On-demand",
       "week-month": "Between once per week and once per month",
-      "month-6month": "Between once per month and once every 6 months"
+      "month-6month": "Between once per month and once every six months",
+      "lt-1hour": "Less then one hour",
```
[the remark this answers is not in the document]
Comment (duke-b) [reply backstage-plugin/plugins/open-dora/src/locales/en/translation.json:13]: ah quite

File: yarn.lock
Comment (kylejwatson) [inline yarn.lock:1]: I think this is also generated by an npm install, sorry I missed it first time (reviewing from my phone)

File: backstage-plugin/plugins/open-dora/src/locales/en/translation.json [hunk 2]
```diff
@@ -9,7 +9,11 @@
       "lt-6month": "Fewer than once per six months",
       "on-demand": "On-demand",
       "week-month": "Between once per week and once per month",
-      "month-6month": "Between once per month and once every 6 months"
+      "month-6month": "Between once per month and once every six months",
+      "lt-1hour": "Less than one hour",
+      "lt-1week": "Less than one week",
+      "week-6month": "Between one week and six months",
+      "mt-6month": "More then six months"
```
Comment (kylejwatson) [inline backstage-plugin/plugins/open-dora/src/locales/en/translation.json:16]:
```suggestion
      "mt-6month": "More than six months"
```

**Review by kylejwatson — APPROVED** [review]
Looks good 👍

**Pull request merged**